src/main/java/Calculator.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Scanner;
+public class Calculator {
+    String productList = "";
+    double sum = 0;
+
+    void calculatorStart() { //начало работы калькулятор
+        int peopleCount;
+        numPeopleCheck check = new numPeopleCheck();
+        peopleCount = check.checkStart();
+        Scanner scanner = new Scanner(System.in);
+        double price;
+        boolean flag = true;// флаг, который позволяет при некорректном вводе стоимости снова ввести
+                                            //стоимость товара, не вводя  снова его название
+        while(true) {
+            if (flag) {
+               getName();
+            }
+
+            System.out.println("Введите стоимость товара в формте '00,00':");
```

Comment:
👏 Классно, что подсказываешь пользователю, в каком формате ожидаются данные!
Закралась небольшая опечатка:
```suggestion
            System.out.println("Введите стоимость товара в формате '00,00':");
```